docs/notebook/export_data.rst

```diff
@@ -0,0 +1,104 @@
+.. _export_data:
+
+***********************************
+Export Data From Jdaviz to Notebook
+***********************************
+
+Specific instructions on exporting data from Jdaviz to your notebook vary slightly for each instance of Jdaviz, including :ref:`specviz-notebook`, :ref:`cubeviz-notebook`, :ref:`mosviz-notebook`, and Imviz.  In all instances, however, you can export data products from the Jdaviz into your notebook and, ultimately, save those variables locally.
+
+Export Position Regions (Imviz)
+-------------------------------
+
+In `Imviz`, you can extract positional regions::
+
+    regions=myinstance.get_interactive_regions()
```

Comment:
This comment applies to all similar occurences. Should the example more closely match the actual example notebooks?

```suggestion
    regions = viz.get_interactive_regions()
```